src/app/controllers/bandAlbumsController.js

```diff
@@ -0,0 +1,29 @@
+export function bandAlbumsController($scope, $routeParams, $location, ApiService) {
+    $scope.bandAlbumsCtrl = this;
+
+    ApiService.getAlbums($routeParams.bandId)
+    .then(function(response) {
+        this.albums = response;
```

Comment:
You should define this field as an empty array before calling the service method. So ng-repeat receives an empty array instead of undefined data before the fetching is successful.